src/lib.rs

```diff
@@ -7,7 +7,7 @@ mod fmt;
 use proc_macro::TokenStream;
 use syn::{parse_macro_input, DeriveInput};
 
-#[proc_macro_derive(Display)]
+#[proc_macro_derive(Display, attributes(display))]
```

Comment:
we probably need to add `source` as one of the attributes this macro is allowed to see.

Reply:
It looks ```source``` was not necessary, since tests are passing without adding the attribute.